mathext/ell_complete.go

```diff
@@ -0,0 +1,165 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import (
+	"math"
+)
+
+// CompleteK computes the complete elliptic integral of the 1st kind, 0≤m≤1:
+//
+//	K(m) = ∫dθ/√(1-m∙sin²(θ)), θ from 0 to π/2.
+//
+// See: Toshio Fukushima, Precise and fast computation of complete elliptic integrals by
+// piecewise minimax rational function approximation,
+// Journal of Computational and Applied Mathematics, Volume 282, 2015, Pages 71-76.
+//
+// DOI: http://dx.doi.org/10.1016/j.cam.2014.12.038
```

Comment:
This is a link, not a DOI. Please just have the link instead of the full reference, "See http://dx.doi.org/10.1016/j.cam.2014.12.038."